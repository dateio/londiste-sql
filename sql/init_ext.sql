\set ECHO none

set log_error_verbosity = 'terse';
set client_min_messages = 'warning';

create extension pgq;
create extension pgq_node;

\set ECHO all

create extension londiste;
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'londiste';

