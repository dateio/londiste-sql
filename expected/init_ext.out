\set ECHO none
create extension londiste;
select array_length(extconfig, 1) as dumpable from pg_catalog.pg_extension where extname = 'londiste';
 dumpable 
----------
        4
(1 row)

